runtime/dotnet/azurewebapp/Microsoft.BotFramework.Composer.WebApp.csproj

```diff
@@ -16,18 +16,18 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.2" />
-    <PackageReference Include="Microsoft.Bot.Builder" Version="4.9.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.9.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.9.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.ApplicationInsights" Version="4.9.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Azure" Version="4.9.1" />
-	<PackageReference Include="Microsoft.Bot.Builder.Dialogs.Declarative" Version="4.9.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Adaptive" Version="4.9.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Debugging" Version="4.9.1-preview" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.ApplicationInsights.Core" Version="4.9.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.9.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.9.1" />
-    <PackageReference Include="Microsoft.Bot.Connector" Version="4.9.1" />
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.9.2" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.9.2" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.9.2" />
+    <PackageReference Include="Microsoft.Bot.Builder.ApplicationInsights" Version="4.9.2" />
+    <PackageReference Include="Microsoft.Bot.Builder.Azure" Version="4.9.2" />
+	<PackageReference Include="Microsoft.Bot.Builder.Dialogs.Declarative" Version="4.9.2" />
```

Comment:
missing space

Reply:
Or extra space depending on the editor :)

Reply:
Good catch, fixed :)